blog/2023-06-16-inline-sdk.mdx

```diff
@@ -0,0 +1,1104 @@
+                        {!playerState.isMuted ? (
+                            <BiVolumeFull color="white" />
+                        ) : (
+                            <BiVolumeMute color="white" />
+                        )}
+                    </button>
+                </div>
+            </div>
+        </div>
+    );
+};
+```
+
+With the band created lets add some styling. Add this to the `src/bands/VideoBand/Band.css` file:
+
+```css title="src/bands/VideoBand/Band.css"
+.video-band.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 2rem;
+    width: 100%;
+}
+
+.video-band h1 {
+    color: white;
+}
+
+.video-band video {
+    width: 100%;
+}
+
+.video-band .video-wrapper {
+    width: 100%;
+    max-width: 700px;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    overflow: hidden;
+    border-radius: 10px;
+}
+
+.video-band .video-wrapper:hover .controls {
+    transform: translateY(0%);
+}
+
+.video-band .controls {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    position: absolute;
+    bottom: 30px;
+    padding: 14px;
+    width: 100%;
+    max-width: 500px;
+    flex-wrap: wrap;
+    background: rgba(255, 255, 255, 0.25);
+    box-shadow: 0 8px 32px 0 rgba(255, 255, 255, 0.1);
+    backdrop-filter: blur(4px);
+    -webkit-backdrop-filter: blur(4px);
+    border-radius: 10px;
+    border: 1px solid rgba(255, 255, 255, 0.18);
+    transform: translateY(150%);
+    transition: all 0.3s ease-in-out;
+}
+
+.video-band .actions button {
+    background: none;
+    border: none;
+    outline: none;
+    cursor: pointer;
+}
+
+.video-band .actions button i {
+    background-color: none;
+    color: white;
+    font-size: 30px;
+}
+
+input[type="range"] {
+    -webkit-appearance: none !important;
+    background: rgba(255, 255, 255, 0.2);
+    border-radius: 20px;
+    height: 4px;
+    width: 350px;
+}
+
+.video-band input[type="range"]::-webkit-slider-thumb {
+    -webkit-appearance: none !important;
+    cursor: pointer;
+    height: 6px;
+}
+
+.video-band input[type="range"]::-moz-range-progress {
+    background: white;
+}
+
+.video-band .velocity {
+    appearance: none;
+    background: none;
+    color: white;
+    outline: none;
+    border: none;
+    text-align: center;
+    font-size: 16px;
+}
+
+.video-band .mute-btn {
+    background: none;
+    border: none;
+    outline: none;
+    cursor: pointer;
+}
+
+.video-band .mute-btn i {
+    background-color: none;
+    color: white;
+    font-size: 20px;
+}
+```
+
+## Deploying the band
+
+### Build the band
+
+Now that the band has been created, we can add it to VertiGIS Studio Web and load it into Inline. In the terminal, run
+
+```
+npm run start
+```
+
+This will start up a development server and create a temporary build of your custom band.
+
+### Adding the band to your app
+
+Open VertiGIS Studio App Designer. In App Designer, open your Inline app.
```

Comment:
"Web Designer"

App Designer is really only an internal name.

Reply:
There are 5 of these on the page.

Reply:
Fixed